gapis/gfxapi/gles/replay.go

```diff
@@ -261,84 +261,88 @@ func (t *destroyResourcesAtEOS) Transform(ctx context.Context, id atom.ID, a ato
 func (t *destroyResourcesAtEOS) Flush(ctx context.Context, out transform.Writer) {
 	id := atom.NoID
 	s := out.State()
-	thread := (uint64)(GetState(s).CurrentThread)
-	c := GetState(s).GetContext(thread)
-	if c == nil {
-		return
-	}
+	for t, c := range GetState(s).Contexts {
```

Comment:
This still does not necessarily free all contexts right?
Let's just create an issue to track this? (or TODO)